### PR TITLE
feat(log): add system log page

### PR DESCRIPTION
**Pull request by oceanlvr**

### CODE REVIEW

File: src/views/log/index.vue [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import Editor from '@/components/Editor.vue'
+import { LogModel } from './types'
+import log from '@/lang/log'
+@Component({
+  components: {
+    Editor,
+  },
+})
+export default class Script extends Vue {
+  @Getter('currentTheme') private theme!: Theme
```
Comment (ysfscream) [inline src/views/log/index.vue:37]: Remove useless Getter

File: src/views/log/index.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator'
+import { Getter, Action } from 'vuex-class'
+import Editor from '@/components/Editor.vue'
+import { LogModel } from './types'
+import log from '@/lang/log'
```
Comment (ysfscream) [inline src/views/log/index.vue:30]: Add a blank line.

File: src/views/log/index.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        renderHighlight="line"
+        disabled="true"
+      />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator'
+import { Getter, Action } from 'vuex-class'
```
Comment (ysfscream) [inline src/views/log/index.vue:27]: Remove useless import package.

File: src/views/log/index.vue [hunk 4]
```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="log-view rightbar">
+    <h1 class="titlebar">
+      {{ $t('log.log') }}
+    </h1>
+    <div
+      class="editor-container log-editor"
+      :style="{
+        height: '80%',
+      }"
```
Comment (0721Betty) [inline src/views/log/index.vue:10]: height is not a variable, why use ":style"

**Review by 0721Betty — APPROVED** [review]

File: src/utils/logger.ts [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+  const STORE_PATH: string = APP.getPath('userData')
+  const LOG_PATH: string = `${STORE_PATH}/logs`
+
+  // In production mode, during the first open application
+  // APP.getPath('userData') gets the path nested and the datastore.js is loaded.
+  // if it doesn't exist, create it.
+  if (!isRenderer) {
+    if (!fs.pathExistsSync(STORE_PATH)) {
+      fs.mkdirpSync(STORE_PATH)
```
Comment (0721Betty) [inline src/utils/logger.ts:18]: Merge into one condition

**Review by 0721Betty — APPROVED** [review]

**Comment by oceanlvr** [comment]
The current appender supports attaching to the console and files

* For console appender, The log of the rendering process will be synchronized to `devtool`, and the main process will be synchronized to the console
* For file appender, the log rendering process will read the latest log file to the editor

image

**Review by ysfscream — APPROVED** [review]

**Pull request merged**